### DISCUSSION TITLE
Semantic release enforces npm v7

### DISCUSSION THREAD

**Discussion by weyert** [semantic-release/semantic-release#2002] [General] [answered]
## Current behavior
I am using `semantic-release` to assist in the versioning of my packages but it looks like that when I am using `npm` within my script definitions it always using `npm v7` even while my system is on v6

An example is:

```
"scripts": {
    "clean": "rm -rf lib && rm -f package-lock.json && rm -rf node_modules",
    "cleaninstall": "npm run clean && npm install",
}
```

## Expected behavior
I would expect I am able to use my shell `npm` in my `scripts` in my `package.json`-file

## Environment

- **semantic-release** version: 17.4.2
- CI environment: Gitlab CI
- Plugins used:     @semantic-release/commit-analyzer, @semantic-release/release-notes-generator, @semantic-release/npm, @semantic-release/gitlab
- **semantic-release** configuration: 
```
{
  "plugins": [
    "@semantic-release/commit-analyzer",
    "@semantic-release/release-notes-generator",
    "@semantic-release/npm",
    "@semantic-release/gitlab",
  ]
}

```
- CI logs: happens locally

**Comment by travi** [comment, marked as answer]
npm is a direct dependency of the npm plugin for semantic-release because it provides the core functionality needed for the publishing process. when npm is installed in the dependency tree of a project, it is the version used in your scripts rather than the one on your global $PATH. 

if you are wanting to use a different version of npm in your project for tasks unrelated to semantic-release, you either need to use a [global installation of semantic-release](https://semantic-release.gitbook.io/semantic-release/usage/installation#global-installation) rather than a local one, or install locally in a script just before releasing on your CI service. either of these options would prevent the version conflict that you are experiencing.

**Reply by weyert** [reply]
Thank you! I think it would be worth to document this. As this is not really what I was expecting spend days trying to find out the cause of this weird behaviour.

**Reply by travi** [reply]
i agree. since you have good context around the problem now that you've worked through it, you may have even better perspective than we do as maintainers. would you be willing to start a PR capturing this information in a docs update in a way that would have helped you better? i'd be happy to provide feedback and help you get it integrated.
